Event/Event.php

```diff
@@ -3,9 +3,11 @@
 namespace Lexik\Bundle\JWTAuthenticationBundle\Event;
 
 use Symfony\Component\EventDispatcher\Event as BaseEvent;
+use Symfony\Component\EventDispatcher\EventDispatcher;
 use Symfony\Contracts\EventDispatcher\Event as ContractsBaseEvent;
+use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
 
-if (class_exists(ContractsBaseEvent::class)) {
+if (is_subclass_of(EventDispatcher::class, EventDispatcherInterface::class)) {
     class Event extends ContractsBaseEvent
```

Comment:
Type error in https://github.com/symfony/symfony/blob/fba11b4dc34e7c589d8c30d5b6a090387d52e648/src/Symfony/Component/EventDispatcher/Debug/TraceableEventDispatcher.php#L314

Reported in this repo https://github.com/lexik/LexikJWTAuthenticationBundle/issues/647 though I'm not sure if something can be improved in Symfony.